docs-kits/kits/Connector Kit/Adoption View/policies-in-catena.md

```diff
@@ -0,0 +1,247 @@
+---
+sidebar_position: 3
+description: 'Conventions and behavior specific to Catena-X'
+title: Policies in Catena-X
+id: connector_kit_adoption_view_policies_cx
+---
+
+## Data Sovereignty in Catena-X
+
+In Catena-X, there's a set of conventions with regard to how Providers structure their `odrl:Offer`s.
+
+A general recommendation is to focus on `permission` property to further specify the contracts' details. In general,
+every data provider can decide on his or her own under which conditions their data datasets (assets) are shared in the
+network. In practice, this works as long as both parties, Provider and Consumer, have the same understanding of its
+legal meaning. Therefore, standardized such `Constraint`s with their `leftOperand`s and `rightOperand`s are key for
+automation. Still, individual freedom of contract is a very high good and is still possible.
+
+This guidance is however also relevant for Enablement Service Providers building components enabling connectivity to the
+Dataspace (as specified in CX-0018). The authoritative resource for schemas is
+the [Catena-X ODRL Profile](https://github.com/catenax-eV/cx-odrl-profile).
+
+As mentioned in the primer on policies, Providers and Consumers must have a common
+understanding of the meaning and consequences of `odrl:Offers` and, on a more granular level, their `odrl:Constraints`.
+That's why there is a set of predefined `odrl:Constraints` - all of which have to
+be [accepted explicitly](working-with-policies.md#consumer-side-odrloffer-in-a-contractrequestmessage) and
+some [checked against a Consumer's VP](working-with-policies.md#provider-side-checking-a-consumers-verifiable-presentation)
+additionally. They are formalized in the [Catena-X ODRL profile](https://github.com/catenax-eV/cx-odrl-profile)
+which extends the regular [ODRL vocabulary](https://www.w3.org/TR/odrl-vocab/). Since usage or contract policies are
+highly dependent on the use case,
+they are described by them in their associated KITs and only general elements are explained in the following.
+
```

Comment:
As with R24.05 the Policy hub comes into play, we might want to add some information regarding it?